pkgs/applications/networking/instant-messengers/dorion/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  lib,
+  dpkg,
+  fetchurl,
+  autoPatchelfHook,
+  wrapGAppsHook,
+  webkitgtk,
+  glib-networking,
+  libappindicator,
+  libayatana-appindicator,
+  gst_all_1,
+}:
+stdenv.mkDerivation rec {
+  name = "dorion";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/SpikeHD/Dorion/releases/download/v${version}/Dorion_${version}_amd64.deb";
+    sha256 = "sha256-IYqJ5mz+XGHf4GVSW2Mq/z8xWLs4Y4KRWZ6fAtIg2tk=";
+  };
+
+  runtimeDependencies = [
```

Comment:
Is this used anywhere?

Reply:
Hmm what do you mean? The hash?

Reply:
The `runtimeDependencies`, I say.

Maybe some dark magic about patchelf...

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/hooks/autopatchelf.section.md
> You can also specify a runtimeDependencies variable which lists dependencies to be unconditionally added to rpath of all executables. This is useful for programs that use dlopen 3 to load libraries at runtime.

Reply:
I need to update my Emacs searching scripts, I suppose.